From d2ee135a3c97f714e6da59adf45a91a2f1632057 Mon Sep 17 00:00:00 2001
From: Keith Holman <Keith.Holman@windriver.com>
Date: Wed, 4 Jun 2014 11:30:53 -0400
Subject: [PATCH] keystone: remove git commands in tests

Keystone tests are designed to be used during development
and will use git to download the latest keystoneclient from
source to test against.  However, on the system installation
we install keystone-client as a separate package, and do not
want to download an external version to test.  In order to
test against the version installed as a separate package,
remove the git calls that clone the keystone-client repository
and just returned the desired directory.

Signed-off-by: Keith Holman <Keith.Holman@windriver.com>
---
 keystone/tests/core.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/keystone/tests/core.py b/keystone/tests/core.py
index 860f68d..9aa54db 100644
--- a/keystone/tests/core.py
+++ b/keystone/tests/core.py
@@ -109,6 +109,7 @@ def checkout_vendor(repo, rev):
 
     working_dir = os.getcwd()
     revdir = os.path.join(VENDOR, '%s-%s' % (name, rev.replace('/', '_')))
+    """
     modcheck = os.path.join(VENDOR, '.%s-%s' % (name, rev.replace('/', '_')))
     try:
         if os.path.exists(modcheck):
@@ -129,6 +130,7 @@ def checkout_vendor(repo, rev):
             fd.write('1')
     except environment.subprocess.CalledProcessError:
         LOG.warning(_('Failed to checkout %s'), repo)
+    """
     cd(working_dir)
     return revdir
 
-- 
1.9.3

